examples/create-react-app-typescript/test/upload-and-check-recording.js

```diff
@@ -0,0 +1,66 @@
+const assert = require("assert");
+const { WebSocket } = require("ws");
+const { SimpleProtocolClient } = require("@replayio/protocol");
+const replay = require("@replayio/replay");
+
+(async () => {
+  try {
+    const apiKey = process.env.REPLAY_API_KEY;
+    const recordings = replay.listAllRecordings();
+    assert(recordings.length === 1, `Expected 1 recording but found ${recordings.length}`);
+    const recordingId = recordings[0].id;
+    console.log(`Uploading recording ${recordingId}`);
+    await replay.uploadRecording(recordingId, { apiKey });
+    console.log("Checking metadata");
+    const metadata = await getTestMetadata(recordingId, apiKey);
+    assert(metadata.test.tests[0].events.main.length > 0, "No test events found in metadata");
```

Comment:
Should we be checking this too?
```suggestion
    assert(
      metadata.test.tests.length > 0 &&
      metadata.test.tests[0].events.main.length > 0,
       "No test events found in metadata"
     );
```

I guess it would fail either way but with a less meaningful error message.